spec/policies/cases/overturned_ico/foi_policy_spec.rb

```diff
@@ -0,0 +1,101 @@
+require 'rails_helper'
+
+RSpec.describe Case::OverturnedICO::FOIPolicy do
+  subject { described_class }
+
+  let(:managing_team)         { find_or_create :team_dacu }
+  let(:other_managing_team)   { create :managing_team }
+  let(:manager)               { managing_team.managers.first }
+  let(:responding_team)       { create :responding_team }
+  let(:responder)             { responding_team.responders.first }
+  let(:dacu_disclosure)       { find_or_create :team_dacu_disclosure }
+  let(:disclosure_specialist) { dacu_disclosure.approvers.first }
+
+  let(:unassigned_case)       { create :overturned_ico_foi }
+
+  let(:manager)               { managing_team.managers.first }
+  let(:other_manager)         { other_managing_team.managers.first }
+  let(:responder)             { responding_team.responders.first }
+  let(:press_officer)         { find_or_create :press_officer }
+  let(:private_officer)       { find_or_create :private_officer }
+  let(:disclosure_approver)   { dacu_disclosure.approvers.first }
+
+
```

Comment:
We discussed that a test that ensures that `Case::OverturnedICO::FOIPolicy` inherits from `Case::FOI::StandardPolicy` would mean we don't need as much testing throughout this test.